### PR TITLE
Create a new task to run E2E API test script

### PR DESCRIPTION
**Pull request by Kinshuk1993**
Create a new task to run the E2E API test script:
- A new task to run the E2E API test script
- The deploy task for each build waits until all resources are up
- The test script only does the test and fails the whole pipeline if it fails

### CODE REVIEW

File: products/bash/release-mq.sh [hunk 1]
```diff
@@ -116,4 +120,87 @@ spec:
...
+  # -------------------------------------- INSTALL JQ ---------------------------------------------------------------------
+
+  echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+  echo -e "\nINFO: Checking if jq is pre-installed..."
+  jqInstalled=false
+  jqVersionCheck=$(jq --version)
+
+  if [ $? -ne 0 ]; then
+  jqInstalled=false
```
Comment (James-Kirk) [inline products/bash/release-mq.sh:132]: Indentation

File: products/bash/release-ace-integration-server.sh [hunk 2]
```diff
@@ -67,3 +71,89 @@ spec:
...
+# -------------------------------------- INSTALL JQ ---------------------------------------------------------------------
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+echo -e "\nINFO: Checking if jq is pre-installed..."
+jqInstalled=false
+jqVersionCheck=$(jq --version)
+
+if [ $? -ne 0 ]; then
+ jqInstalled=false
```
Comment (James-Kirk) [inline products/bash/release-ace-integration-server.sh:85]: Indentation

File: DrivewayDentDeletion/Operators/test-api-e2e.sh [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+# -------------------------------------- INSTALL JQ ---------------------------------------------------------------------
+
+echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
+
+echo -e "\nINFO: Checking if jq is pre-installed..."
+jqInstalled=false
+jqVersionCheck=$(jq --version)
+
+if [ $? -ne 0 ]; then
+ jqInstalled=false
```
Comment (James-Kirk) [inline DrivewayDentDeletion/Operators/test-api-e2e.sh:61]: Indentation

File: products/bash/release-mq.sh (hunk 1 above)
Comment (Kinshuk1993) [reply products/bash/release-mq.sh]: Done

File: products/bash/release-ace-integration-server.sh (hunk 2 above)
Comment (Kinshuk1993) [reply products/bash/release-ace-integration-server.sh]: Done

File: DrivewayDentDeletion/Operators/test-api-e2e.sh (hunk 3 above)
Comment (Kinshuk1993) [reply DrivewayDentDeletion/Operators/test-api-e2e.sh]: Done

**Pull request merged**